src/events.ts

```diff
@@ -59,12 +59,21 @@ export type DeviceDeletedEvent = CommonDeviceEvent<"device.deleted">
 
 export type DeviceThirdPartyIntegrationDetected =
   CommonDeviceEvent<"device.third_party_integration_detected">
+export type DeviceThirdPartyIntegrationNoLongerDetected =
+  CommonDeviceEvent<"device.third_party_integration_no_longer_detected">
+
 export type DeviceSaltoPrivacyModeActivated =
   CommonDeviceEvent<"device.salto.privacy_mode_activated">
 
 export type DeviceSaltoPrivacyModedeactivated =
   CommonDeviceEvent<"device.salto.privacy_mode_deactivated">
 
+export type DeviceConnectionBecameFlaky =
+  CommonDeviceEvent<"device.connection_became_flaky">
```

Comment:
`device.flaky_connection_detected` or `device.unstable_connection_detected` if the pair is `device.connection_stabilized`?

Reply:
i think `connection_became_flaky` is ok, `_detected` to me implies something more definitive (flaky is a judgement call on a degraded system)

Potentially `device.connection_degraded`?

Reply:
I'm think `connection_became_flaky` is still my favorite, I would need to look up what a degraded connection is but not a flaky one.